crates/ty_ide/src/hover.rs

```diff
@@ -296,9 +575,13 @@ mod tests {
         let test = cursor_test(
             r#"
 
-            def foo(a, b): ...
+            def foo(a, b):
+                """The foo function"""
+                return 0
 
-            def bar(a, b): ...
+            def bar(a, b):
+                """The bar function"""
+                return 1
 
             if random.choice([True, False]):
                 a = foo
```

Comment:
Similarly this doesn't change anything, but I added docs just in case something got smarter